include/LightGBM/config.h

```diff
@@ -155,7 +155,6 @@ struct Config {
   // descl2 = missing values must be encoded as ``np.nan`` (Python) or ``NA`` (CLI), not ``0``
```

Comment:
@jameslamb Please add info about missing values representation in R-package.

Reply:
sorry for the delay! Added in https://github.com/microsoft/LightGBM/pull/3699/commits/302a71c98445cd4cfd4b758a3327b36bc2339ed4